File: Java/Easy/1812.java
class Solution {
    public boolean squareIsWhite(String coordinates) {
        if (coordinates.charAt(0) == 'a' || coordinates.charAt(0) == 'c' || coordinates.charAt(0) == 'e'
                || coordinates.charAt(0) == 'g') {
            if (coordinates.charAt(1) == '2' || coordinates.charAt(1) == '4' || coordinates.charAt(1) == '6'
                    || coordinates.charAt(1) == '8') {
                return true;

            }
            return false;
        } else {
            if (coordinates.charAt(1) == '2' || coordinates.charAt(1) == '4' || coordinates.charAt(1) == '6'
                    || coordinates.charAt(1) == '8') {
                return false;

            }
            return true;
        }
    }
}

File: C/Easy/1812.c
bool squareIsWhite(char *coordinates)
{
    int col = coordinates[0] - 'a';
    int row = coordinates[1] - '1';
    if (col % 2 == 0)
    {
        if (row % 2 == 0)
        {
            return false;
        }
        else
        {
            return true;
        }
    }
    else
    {
        if (row % 2 == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}